github-collaboration/github-collaboration.md

```diff
@@ -90,6 +90,21 @@ In the industry, apps are created by teams of developers rather than individual
     git push origin master
     ```
     - Now your fork is up to date with the original Techtonica/curriculum repo. For the moment, at least.
+ 1. Adding a new remote and Knowing more about remote
+     - The commands presented below let you manage connections with other repositories. The following commands will modify         the repo's /.git/config file. The result of the following commands can also be achieved by directly editing the            ./.git/config file  with a text editor. ```git remote add <name> <url>```
+     - Create a new connection to a remote repository. After adding a remote, you’ll be able to use <name> as a convenient         shortcut for <url> in other Git commands. ```git remote rm <name>```
+    - Remove the connection to the remote repository called <name>.
+     ```git remote rename <old-name> <new-name>```
+    - The git remote command will list previously stored remote connections to other repositories.
+     ```git remote```
+    - Invoking git remote with the -v option will print the list of bookmarked repository names and additionally, the corresponding repository URL.  ```git remote -v```.
+ 
+    - ```git pull origin master```
+       - fetches a copy of the master branch from the original repository, and merges it with the current branch you have checked out.
+    - ```git pull```
+      - by default merges your checked out, local branch with the remote branch you created your local branch from.
+
+    - In terms of use-cases, I imagine you would use the 'git pull origin master' form when you want to merge your current branch with a different branch than the one you created your local one with originally. If you just want to merge the latest changes on the remote copy of your current branch (the one you used to create your local copy of the branch), you would use the 'git pull' form.
```

Comment:
instead of this use-case part, can you add a few examples of other ways to git pull <remote name> < branch name>? Different remote or different branch?  And what if you forked a repo but wanted to get recent changes from the original repo?

Reply:
Yay, perfect! Can you use ‘’’ backticks for commands please? Also, your <example name> notation earlier was good, can you do it here too?